src/infrastructure/http/exceptions.py

```diff
@@ -69,3 +70,14 @@ def pydantic_validation_errors_handler(_: Request, error: RequestValidationError
         content=jsonable_encoder(response.dict(by_alias=True)),
         status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
     )
+
+
+def sqlalchemy_database_error_handler(_: Request, error: TimeoutError | InvalidPasswordError) -> JSONResponse:
+    """This function is called if the SQLAlchemy database error was raised."""
+    logger.error(error)
```

Comment:
Pylance complains if you don't coerce to a string here:
```suggestion
    logger.error(str(error))
```